### PR TITLE
Add support simple navigation

### PR DESCRIPTION
**Pull request by alabotski**

### CODE REVIEW

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567&bid=15993745)

File: src/main/java/com/rednavis/vaadin/showcase/views/Constants.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.rednavis.vaadin.showcase.views;
+
+/**
+ * @author Aliaksei Labotski.
+ * @since 13/01/20.
+ */
+public class Constants {
```
Comment (AlexOreshkevich) [inline src/main/java/com/rednavis/vaadin/showcase/views/Constants.java]: Codacy Issue found: [The utility class name 'Constants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567)

File: frontend/src/views/login/login-view.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  static get template() {
+    return html`
+            <div id="header">Login View</div>
+            <div id="content">I'm the future Login Form</div>
+            <hr>
+            <div id="footer">(c)RedNavis</div>`;
+  }
+
+  static get is() {
+    return 'login-view';
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567)

File: frontend/src/views/login/login-view.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567)

File: frontend/src/views/login/login-view.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567)

**Comment by AlexOreshkevich** [comment]
please provide some comments cause thats educational and demo project, remove author tag and creation date

Otherwise make sense, thanks

**Comment by alabotski** [comment]
This simple navigation between two routes.
For example @kate-novik can use it for her task https://github.com/rednavis/vaadin-showcase/issues/3

In general, this is a small example of the use of Polymer 3. Which appeared in 14 Vaadin

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567&bid=15993745)

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567&bid=15993745)

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567&bid=15993745)

**Pull request merged**